sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_patch.py

```diff
@@ -29,11 +28,11 @@ class LogsIngestionClient(GeneratedClient):
     """
 
     def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any) -> None:
-        scope = "https://monitor.azure.com//.default"
+        scope = kwargs.pop("credential_scopes", ["https://monitor.azure.com//.default"])
```

Comment:
Would it be possible to set the default scopes in the swagger README.md so we don't even need to pop the kwarg and pass it in?

example: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md?plain=1#L38

Reply:
Ah, didn't know about this. Thanks for the example! I'll set it this way so that the we can remove the content in the patch files.